File: TransCoder_tokenized_test_set_functions/java/MIRROR_CHARACTERS_STRING.java
static String compute ( String str, int n ) {
  String reverseAlphabet = "zyxwvutsrqponmlkjihgfedcba";
  int l = str . length ( );
  String answer = "";
  for ( int i = 0;
  i < n;
  i ++ ) answer = answer + str . charAt ( i );
  for ( int i = n;
  i < l;
  i ++ ) answer = answer + reverseAlphabet . charAt ( str . charAt ( i ) - 'a' );
  return answer;
}

File: TransCoder_tokenized_test_set_functions/cpp/MIRROR_CHARACTERS_STRING.cpp
string compute ( string str, int n ) {
  string reverseAlphabet = "zyxwvutsrqponmlkjihgfedcba";
  int l = str . length ( );
  for ( int i = n;
  i < l;
  i ++ ) str [ i ] = reverseAlphabet [ str [ i ] - 'a' ];
  return str;
}